internal/bootstrap/data/setting.go

```diff
@@ -144,6 +144,7 @@ func InitialSettings() []model.SettingItem {
 		{Key: conf.ForwardDirectLinkParams, Value: "false", Type: conf.TypeBool, Group: model.GLOBAL},
 		{Key: conf.IgnoreDirectLinkParams, Value: "sign,alist_ts", Type: conf.TypeString, Group: model.GLOBAL},
 		{Key: conf.WebauthnLoginEnabled, Value: "false", Type: conf.TypeBool, Group: model.GLOBAL, Flag: model.PUBLIC},
+		{Key: conf.IsTaskPersistant, Value: "true", Type: conf.TypeBool, Group: model.GLOBAL},
```

Comment:
I suggest placing this switch within the configuration file, as modifying it directly in the database can be challenging if this feature disrupts the program's normal operation.

Reply:
Additionally, the configuration file contains distinct settings for each task, allowing for individual customization of each one.

Reply:
Got it, I will update the code.